src/api/apiCore.cpp

```diff
@@ -535,8 +535,15 @@ void TapePlayer::tape_graph_get_result_size(const YAML::Node& yamlNode)
 		&out_count,
 		&out_size_of);
 
-	if (out_count != yamlNode[2].as<int32_t>()) RGL_WARN("tape_graph_get_result_size: out_count mismatch");
-	if (out_size_of != yamlNode[3].as<int32_t>()) RGL_WARN("tape_graph_get_result_size: out_size_of mismatch");
+	auto tape_count = yamlNode[2].as<int32_t>();
```

Comment:
Naming convention seem to not match a lot of code in this file: snake -> camel?
+
Also, tape_count and tape_size_of may be const.

Reply:
Regarding the naming convention, understandable. In terms of const correctness, it is rather not a matter of performace, that is right. It is more about semantics - if something should not be mutable, we simply make it const.